src/LibFTD2XX.jl

```diff
@@ -247,7 +247,7 @@ function Base.isopen(handle::FT_HANDLE)
     try
       status(handle)
     catch ex
-      if ex == FT_INVALID_HANDLE
+      if ex == FT_INVALID_HANDLE || ex == FT_IO_ERROR
```

Comment:
What do the FT docs say about this error code? Does an FT_IO_ERROR always mean that the port will then be closed or is this for our convenience?

Reply:
I see that you rethrow any errors which come back form the diver _apart_ from these. Why are these special?

Is it true that the handle is definitely closed if FT_IO_ERROR is returned? Docs suggest it could be bad function call also (though I assume this would be checked elsewhere).

Reply:
I haven't actually checked: trial and error has merely shown that
 1. `FT_INVALID_HANDLE` is the code you get when the handle refers to a closed port
 2.  `FT_IO_ERROR` is the code you get when the handle refers to a port which is no longer connected to the system (at least on windows with a `USB <-> Serial Converter X` where `X` = `A`, `B` or `C`.

Perhaps I should let the error propagate if the device has been removed? What do you think?

Reply:
Yes I think letting it propagate and trapping it upstream would be a good idea - makes the library part less 'magical'